### DISCUSSION TITLE
Reference for enzyme turnover (T/2)

### DISCUSSION THREAD

**Discussion by teutonicod** [Open-Systems-Pharmacology/Forum#760] [Questions & Problems]
Hi community,

Do you know if there is a reference for the enzymes turnover values included in PK-Sim? 
I am referring to the T/2 values in liver and intestine.
I would be interested in the reference for CYP3A4, 1A2, 2D6, 2C8/9/19 and 2B6.
Also, are the enzymes regenerated after their degradation? I.e. does the abundance remain constant with time?

Thank you very much for your help!

Donato

**Comment by StephanSchaller** [comment]
Hi Donato,

I can spontaneously answer the second part: Yes, the enzyme levels are regenerated (e.g., after mechanism-based inactivation):

![image](https://user-images.githubusercontent.com/25477643/84250643-979fa200-ab0c-11ea-8367-a04699d48ac9.png)

and 

![image](https://user-images.githubusercontent.com/25477643/84250793-cb7ac780-ab0c-11ea-89e5-426a59241375.png)

where k_deg would be dependent on the T1/2.

Best,
Stephan

**Comment by arketkar** [comment]
Hi Donato,

Few references, though not related to the values used in PK-Sim.

(1) http://dmd.aspetjournals.org/content/suppl/2014/09/18/dmd.114.058404.DC1
(2) https://www.diva-portal.org/smash/get/diva2:170067/FULLTEXT01.pdf

Regards.
Anant

**Comment by teutonicod** [comment]
@arketkar @StephanSchaller , thanks  a lot for your replies!

@sfrechen; @TWendl, any idea about the references for the turnover in PK-Sim?

Thanks in advance!

Donato
